prisma/schema.prisma

```diff
@@ -28,6 +28,7 @@ model Recipe {
   isPublic    Boolean  @default(false)
   author      User     @relation(fields: [authorId], references: [id], onDelete: Cascade, onUpdate: Cascade)
   authorId    Int
+  imagesUrls  String[] @unique
```

Comment:
hmm, why `@unique`?

Reply:
also I think it should be optional so that we do not require user to upload image recipe

Reply:
Unique because url has file name which was generated by `uuidv4`

Prisma automatically initializes `imagesUrls` so there is no need to pass it for e.g. in create request.